### PR TITLE
Minor test cleanup

### PR DESCRIPTION
**Pull request by danmosemsft**
Looked through our disabled tests and noticed a couple nits.

### CODE REVIEW

File: src/System.Runtime/tests/System/Reflection/MemberInfoTests.netcoreapp.cs [hunk 1]
```diff
@@ -337,7 +337,7 @@ public static void HasSameMetadataDefinitionAs_CornerCase_ArrayMethods()
             );
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsWindows))]
+        [PlatformSpecific(TestPlatforms.Windows)]
```
Comment (hughbe) [inline src/System.Runtime/tests/System/Reflection/MemberInfoTests.netcoreapp.cs]: Does this no longer have a fact?

File: src/System.Diagnostics.Process/tests/ProcessTests.cs [hunk 2]
```diff
@@ -58,7 +58,6 @@ private void AssertNonZeroWindowsZeroUnix(long value)
             }
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
         [PlatformSpecific(TestPlatforms.Windows)]  // Expected behavior varies on Windows and Unix
```
Comment (hughbe) [inline src/System.Diagnostics.Process/tests/ProcessTests.cs]: ditto

Comment (danmosemsft) [inline src/System.Diagnostics.Process/tests/ProcessTests.cs]: Unghh

**Comment by danmosemsft** [comment]
@dotnet/dnceng what happened here -- the jobs failed but the details links are 404?

**Comment by danmosemsft** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@danmosemsft @weshaggard 's PR must have gotten merged, which deleted the old jobs.  There used to be a grace period of about a day, but it's not on this server right now.

**Comment by danmosemsft** [comment]
OK. @dotnet-bot test this please

**Comment by danmosemsft** [comment]
@dotnet/dnceng what does it mean when the link from "linux x64 release  build" just goes to a page that never populates? That's not a deleted leg.
https://mc.dot.net/#/user/danmosemsft/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/9bc625bd5da2e498b767c151138155d93ddd8aa0

**Comment by MattGal** [comment]
@danmosemsft the API endpoint is taking forever then timing out in an HTTP 502, specifically this call:

jobs?filter.build=9bc625bd5da2e498b767c151138155d93ddd8aa0&filter.source=pr%2Fjenkins%2Fdotnet%2Fcorefx%2Fmaster%2F&filter.type=test%2Ffunctional%2Fcli%2F&groupBy=job.build&maxResultSets=1

Is it possible you've requeued this one a bunch of times?

**Comment by Chrisboh** [comment]
@alexperovich do you mind taking a look and seeing what is going on here?

**Comment by alexperovich** [comment]
@Chrisboh looking now

**Comment by alexperovich** [comment]
Yes there are 80 jobs in the database for this build. The sheer amount of data that the api needs to aggregate from the database makes the request take longer than the fixed timeout in azure app services of ~2 minutes. The fact that the UI doesn't inform you about the error is also an issue. Filed dotnet/core-eng#974 to track.

**Comment by danmosemsft** [comment]
@dotnet/dnceng the 4 ARM builds are not starting -- it just says triggered. What's up?

**Comment by mmitche** [comment]
@dotnet/dnceng For visibility
@weshaggard Do these jobs appear in the new CI (with the same name)?  If not, then these names would have been orphaned as wes did his update

**Comment by weshaggard** [comment]
>@weshaggard Do these jobs appear in the new CI (with the same name)? If not, then these names would have been orphaned as wes did his update

Those names have been orphaned and can be ignored.

**Comment by danmosemsft** [comment]
@stephentoub can merge? All the jobs in a regular CI run are green here The others are all defunct jobs.

**Pull request merged**